packages/mui-material-next/src/CircularProgress/CircularProgress.tsx

```diff
@@ -38,12 +44,12 @@ const circularDashKeyframe = keyframes`
   }
 `;
 
-const useUtilityClasses = (ownerState) => {
-  const { classes, variant, color, disableShrink } = ownerState;
+const useUtilityClasses = (ownerState: CircularProgressOwnerState) => {
+  const { classes, variant = 'indeterminate', color = 'primary', disableShrink } = ownerState;
 
   const slots = {
     root: ['root', variant, `color${capitalize(color)}`],
-    svg: ['svg'],
+    svg: ['svg', `svg${capitalize(variant)}`],
```

Comment:
```suggestion
    svg: ['svg'],
```

Let's keep the scope of the PR to only convert to TS